src/DataFixtures/AppFixtures.php

```diff
@@ -67,6 +68,8 @@ private function loadTags(ObjectManager $manager): void
 
     private function loadPosts(ObjectManager $manager): void
     {
+        /** @var User $author */
+        /** @var array<Tag> $tags */
```

Comment:
Updated, but to satisfy PHPStan I had to add PHPDocs:
```php
/** @var User $user */
/** @var Tag $tag */
```
for `$this->getReference()` calls, because return value of this method is defined in this way:
```php
@return object
```